### ISSUE TITLE
Learning rate scheduler page shows incorrect figures

### ISSUE THREAD

**Issue by sxjscience** [apache/incubator-mxnet#17291]
## Description
I came across this issue when checking our official guide of learning rate scheduler. https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/training/learning_rates/learning_rate_schedules.html 

The figures are not correct and they all point to the cosine learning rate scheduler. We should correct it. Also, we can ask the users to go to D2L to learn learning rate scheduler: http://d2l.ai/chapter_optimization/lr-scheduler.html
